### DISCUSSION TITLE
Negative Values after NVS Error when flash is full/expired

### DISCUSSION THREAD

**Discussion by loca5790** [emporia-vue-local/esphome#282] [Q&A]
When I get a power cycle on my emporia vues that were originally set to really high flash intervals I get large negative values reported that siginificantly impacts my long term energy monitoring.  I thought HA was capable of only detecting increasing values and resets... anyone have any work arounds?

**Comment by flaviut** [comment]
Is your clock sync working correctly on your device? You can try changing the `time:` field in your configuration like this: https://github.com/emporia-vue-local/esphome/discussions/120#discussioncomment-5275615

**Reply by loca5790** [reply]
I checked and I do not have time configured that way per the linked comment.  The interesting thing is that on power cycles it randomly jumps which sensor acts up.  It usually water heater but it can be negative or positive and it can move from one sensor to another.  It is not every sensor.  It has not bothere me enough to investigate it deeply.  It's on my list of things to flash but..... feel its more risky to flash it OTA via the windows CLI than its worth for a potential.
